src/lib/templateengine.js

```diff
@@ -216,7 +216,8 @@ function TemplateEngine( undefined ) {
   this.bindClickToWidget = false;
     
   // threshold where the mobile.css is loaded
-  this.maxMobileScreenWidth = 480;
+	// this.maxMobileScreenWidth = 480; // modified for Menu Plugin!
+  this.maxMobileScreenWidth = 780;
```

Comment:
Do you really need that? Woudn't a config file with the attribute `max_mobile_screen_width` set in the `<pages>` element be sufficient?

When you do need it: please don't keep the old version in the comment in the line above (219)